packages/cardano-services-client/scripts/createVersionSource.js

```diff
@@ -24,13 +24,24 @@ const apiVersion = Object.fromEntries(
 
 const versionFile = path.join(__dirname, '../version.json');
 const create = () => {
-  const version = JSON.stringify(apiVersion, null, 2);
+  let versionsData = {};
+  if (fs.existsSync(versionFile)) {
+    const data = fs.readFileSync(versionFile, { encoding: 'utf8' });
+    versionsData = JSON.parse(data);
+  }
+  for (const key of Object.keys(apiVersion)) {
+    if (!versionsData[key]) {
+      versionsData[key] = [apiVersion[key]];
+    } else if (!versionsData[key].includes(apiVersion[key])) {
+      versionsData[key].push(apiVersion[key]);
+    }
+  }
```

Comment:
```suggestion
  Object.entries(apiVersion).forEach(({key, value}) => {
    const versionsValue = versionsData[key] ?? [ ];
    versionsData[key] = !versionsValue.includes(apiVersion[key]) 
      ? [...versionsValue, value]
      : versionsValue
  })
```

Reply:
I wrote that first, but linter rules in this repository forbid `Object…forEach` :eyes:

Reply:
:facepalm:

Reply:
I think something like this should work too
```suggestion
  for (const {key, value} of Object.entries(apiVersion)) {
    const versionsValue = versionsData[key] ?? [ ];
    versionsData[key] = !versionsValue.includes(apiVersion[key]) 
      ? [...versionsValue, value]
      : versionsValue
  }
```

Reply:
Fixed in 99e3c4d91fc6f7079a2509553b3e3977efd234d5